src/index.js

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import { createRoot } from 'react-dom/client';
 import './style.css';
-import App from './app';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Home from './pages/Home';
+import Calculator from './pages/Calculator';
+import Quote from './pages/Quote';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
+const root = createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <BrowserRouter>
+    <Routes>
+      <Route path="/math-magicians" element={<Navbar />}>
+        <Route index element={<Home />} />
+        <Route path="/math-magicians/Calculator" element={<Calculator />} />
+        <Route path="/math-magicians/Quote" element={<Quote />} />
+      </Route>
+    </Routes>
+  </BrowserRouter>,
```

Comment:
- **[OPTIONAL]** Instead of deleting the App.js component, I think it would have been better to keep the file and implement this routing there to keep things more modular and simple.